### DISCUSSION TITLE
Improve add-on/extension descriptions (with the goal of increasing adoption)

### DISCUSSION THREAD

**Discussion by make-github-pseudonymous-again** [uBlockOrigin/uBlock-issues#2010] [General]
Is there any chance applying the following patch on descriptions would draw in more non-technical users?

```diff
-uBlock Origin is not an "ad blocker", ...
+uBlock Origin is more than an "ad blocker", ...
```

**Comment by u-RraaLL** [comment]
Did you mean to write:

> uBlock Origin is far beyond a mere "ad blocker" (...)

?

**Comment by gorhill** [comment]
I really want to stick to the fact that uBO is **not** an ad blocker, even though it can function as one by the fact that it mostly compatible with EasyList syntax. "Adoption" is not what motivate me to work on uBO and I rather this not being used to argue for changes.

**Reply by make-github-pseudonymous-again** [reply]
Thanks. Case closed on my side.
